validphys2/src/validphys/scalevariations/scalevariationtheoryids.yaml

```diff
@@ -353,3 +353,35 @@ scale_variations_for:
       (2, 0.5): 41_100_026
       (2, 1): 41_100_027
       (2, 2): 41_100_028
+
+  # N3LO: MHOU + IHOU
+  - theoryid: 40_005_000
+    variations:
+      (0, 0, 0, 0, 0, 0, 0): 40_005_000 # central th
+      (1, 0, 0, 0, 0, 0, 0): 40_005_001 # anomalous dimensions variations
+      (2, 0, 0, 0, 0, 0, 0): 40_005_002
+      (0, 1, 0, 0, 0, 0, 0): 40_005_003
+      (0, 2, 0, 0, 0, 0, 0): 40_005_004
+      (0, 0, 1, 0, 0, 0, 0): 40_005_005
+      (0, 0, 2, 0, 0, 0, 0): 40_005_006
+      (0, 0, 0, 1, 0, 0, 0): 40_005_007
+      (0, 0, 0, 2, 0, 0, 0): 40_005_008
+      (0, 0, 0, 0, 1, 0, 0): 40_005_009
+      (0, 0, 0, 0, 2, 0, 0): 40_005_010
+      (0, 0, 0, 0, 0, 1, 0): 40_005_011
+      (0, 0, 0, 0, 0, 2, 0): 40_005_012
+      (0, 0, 0, 0, 0, 0, 1): 40_005_013
+      (0, 0, 0, 0, 0, 0, 2): 40_005_014
+      (0.5, 1): 40_005_019 # scale variations
+      (2, 1): 40_005_020
+      (0.5, 0.5): 40_005_021
+      (1, 0.5): 40_005_022
+      (2, 0.5): 40_005_023
+      (0.5, 2): 40_005_024
+      (1, 2): 40_005_025
+      (2, 2): 40_005_026
+      (-1, -1): 40_005_015 # DIS coeff functions fact variations
+      (1, 1): 40_005_016
```

Comment:
I changed the naming for some of these things so now these are `lower bound DIS` `upper bound DIS` and `central DIS`. You will now also explicitly have to add the central theory for a given covmat since it may not always be the same (e.g. I may construct an alphas covmat with the central theory being NNLOxQED but the central theory for the MHOU covmat should be just NNLO)